### PR TITLE
TELCODOCS-82 - removing unsupported mirror reg procedure for 4.9 ZTP GA

### PR DESCRIPTION
**Pull request by aireilly**
For main, enterprise-4.9

This update removes an unsupported mIrror registry procedure for 4.9 ZTP GA and reuses the canonical procedure and modules for disconnected mirror registry. 

https://issues.redhat.com/browse/TELCODOCS-82

Preview: https://deploy-preview-36970--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected#ztp-acm-preparing-to-install-disconnected-acm_ztp-deploying-disconnected

### CODE REVIEW

File: scalability_and_performance/ztp-deploying-disconnected.adoc [hunk 1]
```diff
@@ -28,35 +28,49 @@ include::modules/ztp-du-host-networking-requirements.adoc[leveloffset=+1]
...
+** link:https://www.redhat.com/en/technologies/cloud-computing/quay[Red Hat Quay]
+** link:https://jfrog.com/artifactory/[JFrog Artifactory]
+** link:https://www.sonatype.com/products/repository-oss?topnav=true[Sonatype Nexus Repository]
+** link:https://goharbor.io/[Harbor]
+--
+If you have an entitlement to Red Hat Quay, see the documentation on deploying Red Hat Quay link:https://access.redhat.com/documentation/en-us/red_hat_quay/3.5/html/deploy_red_hat_quay_for_proof-of-concept_non-production_purposes/[for proof-of-concept purposes] or link:https://access.redhat.com/documentation/en-us/red_hat_quay/3.5/html/deploy_red_hat_quay_on_openshift_with_the_quay_operator/[by using the Quay Operator]. If you need additional assistance selecting and installing a registry, contact your sales representative or Red Hat support.
+
+include::modules/installation-about-mirror-registry.adoc[leveloffset=+2]
+
+.Additional information
```
Comment (bobfuru) [inline scalability_and_performance/ztp-deploying-disconnected.adoc:57]: We typically use "Additional resources" for nested includes and anchors (see [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-assemblies)).

```suggestion
.Additional resources
```

File: scalability_and_performance/ztp-deploying-disconnected.adoc [hunk 2]
```diff
@@ -28,35 +28,49 @@ include::modules/ztp-du-host-networking-requirements.adoc[leveloffset=+1]
...
+For information on viewing the CRI-O logs to view the image source, see xref:../installing/validating-an-installation.html#viewing-the-image-pull-source_validating-an-installation[Viewing the image pull source].
+
+[id="installing-preparing-mirror_{context}"]
+=== Preparing your mirror host
+
+Before you perform the mirror procedure, you must prepare the host to retrieve content
+and push it to the remote location.
 
-include::modules/cli-installing-cli.adoc[leveloffset=+2]
+include::modules/cli-installing-cli.adoc[leveloffset=+3]
```
Comment (bobfuru) [inline scalability_and_performance/ztp-deploying-disconnected.adoc:67]: This isn't in the scope of this PR, but I am a little curious as to why this procedure (or rather, set of procedures) is included here as a step within the "Preparing your mirror host" procedure. Could it be relocated as one of the prerequisites with a link to the `oc` install docs (https://docs.openshift.com/container-platform/4.9/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli)?

**Review by bobfuru — COMMENTED** [review]
Overall, looks great! A few nits for consideration. Thanks!

File: scalability_and_performance/ztp-deploying-disconnected.adoc (hunk 2 above)
Comment (aireilly) [reply scalability_and_performance/ztp-deploying-disconnected.adoc:67]: I'm just reusing the procedure as documented here: https://docs.openshift.com/container-platform/4.8/installing/installing-mirroring-installation-images.html

The user needs to install `oc` on the mirror host.

**Pull request merged**